packages/core/bootstrap/src/lib/ws/index.ts

```diff
@@ -17,6 +17,9 @@ export const withWebSockets = (store: Store<RootState>) => (makeWsHandler?: Make
   // TODO: Warmer
   const wsHandler = makeWsHandler()
   const wsConfig = getWSConfig()
+  if (wsHandler.init !== undefined) {
```

Comment:
And instead of this, `const wsHandler = await makeWsHandler()` would actually make the initialization

Reply:
Yes, I agree. Let's make `type WSHandlerFactory = () => Promise<WSSubscriptionHandler>` for cases like this. cc @RodrigoAD